pkg/controller/podautoscaler/horizontal.go

```diff
@@ -383,10 +395,12 @@ func (a *HorizontalController) reconcileAutoscaler(hpav1Shared *autoscalingv1.Ho
 
 	rescale := true
 
+	// TODO: conditions for current below/above?
```

Comment:
whoops, that escaped cleanup when it shouldn't have.  I was pondering whether or not to have a condition indicating that the current replica count was outside the scale limits, but I decided against it for now.